src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num !== undefined) {
+      sum += num;
```

Comment:
Ensure that `num` is a number before adding it to `sum`. You might want to check if `num` is of type `number` to prevent unexpected behavior when non-numeric values are passed.